### ISSUE TITLE
Add Focus Ring to Checkbox

### ISSUE THREAD

**Issue by yuderekyu** [cerner/terra-core#1090]
# Issue Description
The checkbox `container-focus` style, specified in several custom themes, has been replaced with a large focus ring surrounding the checkbox.`container-focus` styling should be removed and a `focus-ring` styling should be added.

*Edit: I can still have state management removed and utilize css selectors for the focus ring.
